cmd/rook/main.go

```diff
@@ -232,6 +240,14 @@ func setLogLevel() {
 	capnslog.SetGlobalLogLevel(cfg.logLevel)
 }
 
+func logStartupInfo(cmdFlags *pflag.FlagSet) {
+	// log the version number, arguments, and all final flag values (environment variable overrides
+	// have already been taken into account)
+	flagValues := flags.GetFlagsAndValues(cmdFlags)
+	logger.Infof("starting rook %s with arguments '%s'", version.Version, strings.Join(os.Args, " "))
+	logger.Infof("flag values: %s", strings.Join(flagValues, ", "))
```

Comment:
I'd say the challenge with logs is that they are shared for troubleshooting and secrets aren't intended to be shared in that case. What if we print a dummy value for flags that have `secret` in the name?